README.md

```diff
@@ -112,8 +115,10 @@ A curated list of tools and resources for Platform Engineering.
 - [Platform Engineering at Palo Alto Networks](https://medium.com/engineering-at-palo-alto-networks/platform-engineering-at-palo-alto-networks-part-2-315bd7b0fbfa)
 - [Platform Engineering story from a CTO: WHY, WHAT, HOW](https://medium.com/agorapulse-stories/platform-engineering-part-1-why-the-evolution-of-developer-cognitive-load-9f36f5cc2888)
 - [Create Preview Environments with Terraform, GitHub Actions, and Vercel](https://developer.hashicorp.com/terraform/tutorials/applications/preview-environments-vercel)
+- [The Internal Developer Platform Revolution: what every CTO should know](https://blog.mia-platform.eu/en/the-internal-developer-platform-revolution)
 
 ## Blogs and media platforms
+- [Mia-Platform Blog](https://blog.mia-platform.eu/en)
```

Comment:
Please move to an end (general principle used in the list additions, maybe one day better to switch to an alphabetical or other order, but for now typically new links added in the end of the section)